lib/node/asar-fs-wrapper.ts

```diff
@@ -841,6 +841,27 @@ export const wrapFsWithAsar = (fs: Record<string, any>) => {
     return files;
   };
 
+  const modBinding = internalBinding('modules');
```

Comment:
Refs https://github.com/nodejs/node/pull/50322 - needed to enable correct package.json reads for asar

Reply:
Can we add a comment in the code to reference this?

Reply:
@samuelmaddock we can but essentially every function in this file serves the same purpose - would you just want the file generally to be a little better documented?